src/com/connectsdk/service/NetcastTVService.java

```diff
@@ -2348,19 +2348,23 @@ public void unsubscribe(URLServiceSubscription<?> subscription) {
     protected void updateCapabilities() {
         List<String> capabilities = new ArrayList<String>();
 
+        Collections.addAll(capabilities, MediaPlayer.Capabilities);
```

Comment:
Could you please extract these capabilities change (except Rewind and FF) into a separate commit? Or even remove from this PR.